### PR TITLE
Fix deb dependencies during multiple profile building

### PR DESCRIPTION
**Pull request by salvatore-coppola**
This changes fix the problem with the build when multiple profiles are used.
Before these changes all deb files used the same nvidia-jetson-nano dependencies.

### CODE REVIEW

**Comment by mstankovic** [comment]
seems good to me. There are no more errors related to iptables and named services. All networking tests are passing at least for Rpi3

**Comment by salvatore-coppola** [comment]
@MMaiero this pr could be reviewed

File: kura/distrib/src/main/deb/control-nvidia-jetson-nano/prerm [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```
Comment (MMaiero) [inline kura/distrib/src/main/deb/control-nvidia-jetson-nano/prerm]: Would it be better if we have those files here in the specific target device folder like in here https://github.com/eclipse/kura/tree/develop/kura/distrib/src/main/resources/nvidia-jetson-nano?
What do you think?

Comment (salvatore-coppola) [reply kura/distrib/src/main/deb/control-nvidia-jetson-nano/prerm]: ok, but I need to do another round of tests to be sure there aren't broken paths to those files.

**Review by MMaiero — APPROVED** [review]

**Comment by MMaiero** [comment]
The changes go over 1000 lines, but they are mainly due to the duplication of files needed to keep up with different dependencies for different profiles.

**Pull request merged**